frontend/catalyst/jax_tracer.py

```diff
@@ -106,7 +106,7 @@ def _eval_jaxpr(*args):
 
 KNOWN_NAMED_OBS = (qml.Identity, qml.PauliX, qml.PauliY, qml.PauliZ, qml.Hadamard)
 
-FORCED_ORDER_PRIMITIVES = {qdevice_p, qextract_p, qinst_p}
+FORCED_ORDER_PRIMITIVES = {qdevice_p}
```

Comment:
Could you add a comment either her or in the sort function, to warn that the sort mechanism is very costly for large circuits.

Reply:
Good idea. Thanks!